crates/pop-cli/src/commands/build/mod.rs

```diff
@@ -62,7 +63,11 @@ impl Command {
 		#[cfg(feature = "contract")]
 		if pop_contracts::is_supported(args.path.as_deref())? {
 			// All commands originating from root command are valid
-			BuildContractCommand { path: args.path, release: args.release, valid: true }
+			let release = match args.profile.unwrap_or(Profile::Debug) {
```

Comment:
Do we want `Debug` as default? before it was `--release` the default value: https://github.com/r0gue-io/pop-cli/blob/main/crates/pop-cli/src/commands/build/spec.rs#L138

Reply:
This is for smart contracts

Reply:
You are right, my mistake